src/containers/Transactions/simpleTab.scss

```diff
@@ -87,5 +87,30 @@ $subdued-color: $gray-400;
         @include bold;
       }
     }
+
+    .attestation {
+      margin: 16px 0px;
+      background: rgba(34 37 43 / 50%);
```

Comment:
This should be a variable. `color.adjust($some-color, .5)`

Reply:
This paradigm didn't work - SCSS had no clue what `color` was. It worked when I used `opacity` instead, though.